src/vbk/pop_service_impl.cpp

```diff
@@ -410,8 +390,9 @@ bool evalScriptImpl(const CScript& script, std::vector<std::vector<unsigned char
     return true;
 }
 
-bool parseBlockPopPayloadsImpl(const CBlock& block, const CBlockIndex& pindexPrev, const Consensus::Params& params, BlockValidationState& state, std::vector<altintegration::AltPayloads>* payloads) EXCLUSIVE_LOCKS_REQUIRED(cs_main)
+bool parseBlockPopPayloadsImpl(const CBlock& block, const CBlockIndex& pindexPrev, const Consensus::Params& params, BlockValidationState& state, std::vector<altintegration::AltPayloads>* payloads)
 {
```

Comment:
Are you sure EXCLUSIVE_LOCKS_REQUIRED should be removed?

Reply:
yeah, here it is perfectly fine